collect_app/src/main/java/org/odk/collect/android/utilities/CustomTabHelper.java

```diff
@@ -89,9 +92,13 @@ public CustomTabsServiceConnection getServiceConnection() {
     public void openUri(Context context, Uri uri) {
         if (getPackageName(context).size() != 0) {
             //open in chrome custom tab
-            CustomTabsIntent customTabsIntent = new CustomTabsIntent.Builder().build();
-            customTabsIntent.intent.setPackage(getPackageName(context).get(0));
-            customTabsIntent.launchUrl(context, uri);
+            new CustomTabsIntent.Builder()
+                    .setToolbarColor(ContextCompat.getColor(context, R.color.tintColor))
```

Comment:
I'd prefer not adding the animation and the tabs toolbar highlight. @yanokwa what do you think?

Reply:
I totally didn't notice this, so good catch, @lognaturel!

I think we should be a little defensive when we add things that are visually appealing but don't add real functionality. It looks nice, but it is more code (it all adds up!) and animations often make transitions feel slower. And on low power devices, it might also actually be slower.

If we do want to add these bits of flair there should be a clear reason why and it should be consistent. In this case, it's not clear why we need an animation. Further, we don't use this animation anywhere else. For those reasons, I propose we cut it.

Reply:
👍 
I'll updated the PR with no animations and remove the toolbar color